File: Backend/LogisticsPlatform.Application/DTOs/Customers/CustomerBillingDto.cs
using LogisticsPlatform.Domain.Enums;


namespace LogisticsPlatform.Application.DTOs.Customers
{
    public class CustomerBillingDto
    {
        public CustomerPaymentTerms Terms { get; set; }
        public CustomerPaymentMethod Method { get; set; }
        public decimal CreditLimit { get; set; }
        public bool AutoInvoice { get; set; }
    }

}

File: Frontend/src/app/core/models/customers/customer-billing.dto.ts
import { CustomerPaymentMethod } from "../../enums/customers/customer-payment-method.enum";
import { CustomerPaymentTerms } from "../../enums/customers/customer-payment-terms.enum";


export interface CustomerBillingDto {
    terms: CustomerPaymentTerms;
    method: CustomerPaymentMethod;
    creditLimit: number;
    autoInvoice: boolean;
}
